src/crypto/crypto_ec.cc

```diff
@@ -928,22 +925,22 @@ std::shared_ptr<KeyObjectData> ImportJWKEcKey(
   if (!jwk->Get(env->context(), env->jwk_x_string()).ToLocal(&x_value) ||
       !jwk->Get(env->context(), env->jwk_y_string()).ToLocal(&y_value) ||
       !jwk->Get(env->context(), env->jwk_d_string()).ToLocal(&d_value)) {
-    return std::shared_ptr<KeyObjectData>();
+    return KeyObjectData();
   }
 
   if (!x_value->IsString() ||
       !y_value->IsString() ||
       (!d_value->IsUndefined() && !d_value->IsString())) {
     THROW_ERR_CRYPTO_INVALID_JWK(env, "Invalid JWK EC key");
-    return std::shared_ptr<KeyObjectData>();
+    return KeyObjectData();
   }
 
   KeyType type = d_value->IsString() ? kKeyTypePrivate : kKeyTypePublic;
 
   ECKeyPointer ec(EC_KEY_new_by_curve_name(nid));
   if (!ec) {
     THROW_ERR_CRYPTO_INVALID_JWK(env, "Invalid JWK EC key");
-    return std::shared_ptr<KeyObjectData>();
+    return KeyObjectData();
```

Comment:
Nit, but not so nit: This applies to the previous 4 as well.

```suggestion
    return {};
```